### PR TITLE
DefaultArgParser.py: Adds Default log level to `-L help`

### PR DESCRIPTION
**Pull request by nitinkgp23**
This adds default log level - INFO to `-L help`.

Closes issue https://github.com/coala/coala/issues/3618

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

**After you submit your pull request, DO NOT click the 'Update Branch' button**

### CODE REVIEW

**Comment by nitinkgp23** [comment]
Can someone help me with this? Why are the tests failing?

**Comment by nitinkgp23** [comment]
The maximum no of charcters permitted in shortlog is 50. It will exceed 50 if i do that.

**Comment by Techievena** [comment]
Remove `.py` :smiley:

**Comment by nitinkgp23** [comment]
@Techievena circleci test is failing now. What should i do?

**Comment by Techievena** [comment]
Ask in gitter for maintainers to trigger rebuild. Some issues with Circle CI. Will be fixed with rebuild. Please remove `issues` after `Closes`.

File: coalib/parsing/DefaultArgParser.py [hunk 1]
```diff
@@ -166,7 +166,8 @@ def default_arg_parser(formatter_class=None):
     outputs_group.add_argument(
         '-L', '--log-level', nargs=1,
         choices=['ERROR', 'INFO', 'WARNING', 'DEBUG'], metavar='ENUM',
-        help='set log output level to ERROR/INFO/WARNING/DEBUG')
+        help='set log output level to ERROR/INFO/WARNING/DEBUG' +
+             'default to INFO')
```
Comment (Techievena) [inline coalib/parsing/DefaultArgParser.py]: Please leave a space before `default` or after `DEBUG`

**Comment by nitinkgp23** [comment]
Hey @Techievena, I think I have addressed all your comments. Let's see how Circle CI responds now.

**Comment by Techievena** [comment]
ack  	5d632e3

File: coalib/parsing/DefaultArgParser.py [hunk 2]
```diff
@@ -166,7 +166,8 @@ def default_arg_parser(formatter_class=None):
     outputs_group.add_argument(
         '-L', '--log-level', nargs=1,
         choices=['ERROR', 'INFO', 'WARNING', 'DEBUG'], metavar='ENUM',
-        help='set log output level to ERROR/INFO/WARNING/DEBUG')
+        help='set log output level to ERROR/INFO/WARNING/DEBUG, ' +
+             'default to INFO')
```
Comment (Makman2) [inline coalib/parsing/DefaultArgParser.py]: should be "default**s** to"^^

File: coalib/parsing/DefaultArgParser.py [hunk 3]
```diff
@@ -166,7 +166,8 @@ def default_arg_parser(formatter_class=None):
     outputs_group.add_argument(
         '-L', '--log-level', nargs=1,
         choices=['ERROR', 'INFO', 'WARNING', 'DEBUG'], metavar='ENUM',
-        help='set log output level to ERROR/INFO/WARNING/DEBUG')
+        help='set log output level to ERROR/INFO/WARNING/DEBUG, ' +
```
Comment (Makman2) [inline coalib/parsing/DefaultArgParser.py]: no need for `+`

**Comment by Makman2** [comment]
commit message:
> DefaultArgParser: Default log level to `-L help`

Though `Default` could be treated as a verb (that's what our GitCommitBear thinks in our CI), it is not in this context. You should add a verb to the commit message, like in `Add default log level to help` or so.

**Comment by Makman2** [comment]
unack f724844

**Comment by nitinkgp23** [comment]
I tried this message before "Add default .. ", but this exceeded the 50 character limit of shortlog. Therefore, I had to get rid of the word 'Add'.

**Comment by Makman2** [comment]
@nitinkgp23 "Add" was the wrong word to get rid off ;P
Rather strip detail information, like the `-L help`

**Comment by Makman2** [comment]
commit message, not PR title ;)

**Comment by nitinkgp23** [comment]
@Makman2 I have addressed all the comments. I hope it's fine now.

**Comment by Makman2** [comment]
We use imperative tense (hope it's the correct grammatical term), so `Adds` -> `Add` :)
\+ I would outline more that you add it to help, so this commit message should work:
```
DefaultArgParser: Add default log level to help
```

**Comment by Makman2** [comment]
unack b89a916

**Comment by Makman2** [comment]
should be the last iteration, after your changes we should be able to merge :+1: :)

**Comment by nitinkgp23** [comment]
@Makman2 Did as you said :) . Btw, what does ack and unack mean? I couldn't understand those comments.

**Comment by supergr35** [comment]
ack b2a044

**Comment by supergr35** [comment]
This should hopefully explain the meaning behind ack and unack :) 
http://api.coala.io/en/latest/Developers/Review.html?highlight=review#manual-review-process

**Comment by Makman2** [comment]
@rultor merge

**Comment by nitinkgp23** [comment]
:D Thanks @Makman2 . That was my first PR merged! Hope to have many more soon! :D

**Pull request merged**